client_it_test.go

```diff
@@ -321,3 +321,112 @@ func TestClusterReconnection_ReconnectModeOff(t *testing.T) {
 	time.Sleep(100 * time.Millisecond)
 	assert.Equal(t, false, c.Running())
 }
+
+func TestClient_GetDistributedObjects(t *testing.T) {
+	const (
+		testMapName = "get-distributed-objects-map"
+		testSetName = "get-distributed-objects-set"
+	)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 40*time.Second)
+	defer cancel()
+
+	cls := it.StartNewClusterWithOptions("go-cli-test-cluster", 15701, 1)
```

Comment:
Is there a reason a new cluster is created for this test? How about using `it.Tester` or `it.TesterWithConfigBuilder`? They re-use a pre-existing cluster and would simplify this test.

Reply:
Are we sure all the tests destroy their objects after complete? I couldn't see such a cleanup but if so, and if all the integration tests run serially (not sure if they do), I'll change them to use an existing cluster. Otherwise, we'll not be able to assert the number of objects returned by the new API.